recipes-wigwag/mbed-fcce/mbed-fcce/psa_storage_user_config.h

```diff
@@ -0,0 +1,37 @@
+// ----------------------------------------------------------------------------
+// Copyright 2017-2018 ARM Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------
+
+#ifndef PSA_STORAGE_USER_CONFIG_H
+#define PSA_STORAGE_USER_CONFIG_H
+
+/**
+* \def PSA_STORAGE_FILE_C_STORAGE_PREFIX
+*
+* Define the path to the directory for Internal Trusted Storage
+* (PSA ITS) files representing persisted objects. For example,
+* to store files in "/home/username" define
+* PSA_STORAGE_FILE_C_STORAGE_PREFIX "/home/username/"
+* (note the appended "/").
+*/
+#ifdef PSA_STORAGE_FILE_C_STORAGE_PREFIX
+    #undef PSA_STORAGE_FILE_C_STORAGE_PREFIX
+#endif
+
+#define PSA_STORAGE_FILE_C_STORAGE_PREFIX "/userdata/mbed/psa/"
```

Comment:
```suggestion
#define PSA_STORAGE_FILE_C_STORAGE_PREFIX "/userdata/mbed/mcc_config/"
```
Can we do this? so that is consistent with previous releases and we also won't have to change the CI checks.

Reply:
If yes, then we can remove it from here as well for all platforms - https://github.com/PelionIoT/meta-mbed-edge/blob/add-parsec-support/recipes-connectivity/mbed-edge-core/files/rpi3/target.cmake#L16

Reply:
I have tried changing both patches to /userdata/mbed/mcc_config and this works OK - I provisioned the device and it connected to pelion after.
Something does create a /userdata/mbed/psa folder - it contains a flashSim data file.

Reply:
I think Edge Core is doing that.. I wasn't aware that `PAL_FS_MOUNT_POINT_PRIMARY` doesn't set the location of the provisioning data when trusted storage is enabled. I will raise a patch in edge-core recipe to create the same folder.